app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/multipledisposals/asset_type_for_non_uk_residents.scala.html

```diff
@@ -72,56 +75,78 @@
   }
 }
 
-@options = @{
-    List(
-      RadioOption(
-        label = messages(s"$key.Residential"),
-        content = None,
-        optionHelpText = Some(Html(messages(s"$key.Residential.helpText")))
-      ),
-      RadioOption(
-        label = messages(s"$key.NonResidential"),
-        content = None,
-        optionHelpText = Some(Html(messages(s"$key.NonResidential.helpText")))
-      ),
-      RadioOption(
-        label = messages(s"$key.MixedUse"),
-        content = None,
-        optionHelpText = Some(Html(messages(s"$key.MixedUse$userKey.helpText")))
-      ),
-      RadioOption(
-        label = messages(s"$key$userKey.IndirectDisposal"),
-        content = None,
-        optionHelpText = Some(Html(messages(s"$key.IndirectDisposal.helpText")))
-      )
-    )
-}
-
-@mainTemplate(title = title, userType = request.userType, hasErrors = hasErrors) {
-
-    @backLinkComponent(backLink)
+@selected = @{form.value.map(_.map(toIndex)).getOrElse(List.empty)}
 
-    @if(hasErrors) {
-        @errorSummary(form, Some(s"$key$userKey"))
-    }
+@checkBoxes = @{List(
+  CheckboxItem(
+    id = Some(messages(s"$key-0")),
```

Comment:
there are no messages with these keys - also we should never use `message()` values as form ids - message values are content, subject to change and can be in another language - instead remove this id property and let GovukCheckboxes assign the id in a predictable way - this is why tests are failing - there is no message with this key (or the other "id" message keys used below)